### ISSUE TITLE
Replace git clone by git clone --depth 1 and check out qmc2 completely

### ISSUE THREAD

**Issue by i30817** [VR51/oneup#1]
two problems:
getting the mame source unnecessarily downloads all revisions, using `git clone depth 1` takes care of this.
qmc2 build without existing source for some reason didn't download the last revision for me (sorry don't know svn as much as git). I had to go down to the ~/src/qmc dir and do a svn update on the directory before the script started building otherwise there were plenty of errors because of missing files.

**Comment by VR51** [comment]
Thanks for reporting this.

I was able to build QMC2 yesterday though I only ran the script in update mode and not (clean) install mode. Will test the build process from a clean slate today.

I did notice a download bug with the MAME installer yesterday, which I fixed. The bug had crept in with the previous OneUp! script rebuild. The QMC2 build error might be related.

Will add `depth 1` to the git command. That's a good idea.
